cartography/intel/aws/rds.py

```diff
@@ -169,9 +249,18 @@ def _validate_rds_endpoint(rds):
     return ep
 
 
-def cleanup_rds_instances(neo4j_session, common_job_parameters):
+def _get_db_subnet_group_arn(region, current_aws_account_id, db_subnet_group_name):
+    """
+    Return an ARN for the DB subnet group name by concatenating the account name and region.
+    This is done to avoid another AWS API call since the describe_db_instances boto call does not return the DB subnet
+    group ARN.
+    """
+    return f"arn:aws:{region}:{current_aws_account_id}:{db_subnet_group_name}"
```

Comment:
Format for subnet ARNs is `arn:aws:ec2:region:account-id:subnet/subnet-id`.

Reply:
Great catch and good doc, thanks